src/Commands/LightsOn.cpp

```diff
@@ -22,6 +25,17 @@ void LightsOn::Initialize() {
 
 // Called repeatedly when this Command is scheduled to run
 void LightsOn::Execute() {
+  if (shouldFlash) {
+    if(! flashed){
+      flash.StartPeriodic(0.125);
+      flashed = true;
+    }
+  }
```

Comment:
This block is redundant with the block that starts 
```
   if (Puncher::getInstance()->IsCubeIn())
```

Add `shouldFlash` to the logic to enable flashing.